src/nvme/types.h

```diff
@@ -5305,6 +5330,9 @@ enum nvme_features_id {
 	NVME_FEAT_FID_RESV_MASK					= 0x82,
 	NVME_FEAT_FID_RESV_PERSIST				= 0x83,
 	NVME_FEAT_FID_WRITE_PROTECT				= 0x84,
+
+	NVME_MI_FEAT_CTRL_METADATA	= 0x7E,
+	NVME_MI_FEAT_NS_METADATA	= 0x7F,
```

Comment:
These should follow the name scheme from above and also placed accordingly their value, that is before NVME_FEAT_FID_SW_PROGRESS.